mage_ai/settings/__init__.py

```diff
@@ -19,8 +19,8 @@
 LDAP_BIND_DN = os.getenv('LDAP_BIND_DN', 'cd=admin,dc=example,dc=org')
 LDAP_BIND_PASSWORD = os.getenv('LDAP_BIND_PASSWORD', 'admin_password')
 LDAP_BASE_DN = os.getenv('LDAP_BASE_DN', 'dc=example,dc=org')
-LDAP_AUTHENTICATATION_FILTER = os.getenv('LDAP_AUTHENTICATATION_FILTER',
-                                         '(&(|(objectClass=Pers)(objectClass=gro))(cn={username}))')
+LDAP_AUTHENTICATION_FILTER = os.getenv('LDAP_AUTHENTICATION_FILTER',
```

Comment:
There's a typo in `AUTHENTIFICATION_MODE` and `LDAP_AUTORIZATION_FILTER` as well.